style/style.css

```diff
@@ -744,8 +744,9 @@ label {
   opacity: 1;
 }
 
-.error-email {
-  color: red;
+p.error-email {
+  color: #fff;
   font-size: 16px;
   font-family: 'Poppins', sans-serif;
+  font-weight: 700;
 }
```

Comment:
- [OPTIONAL] It may be a good idea to fix the `width` property of your `.error-email` for desktop size. 😉 

![mobile-form](https://user-images.githubusercontent.com/79876959/141163465-6f982001-acf6-41af-b151-e70218f2c07e.PNG)
![desktop-form](https://user-images.githubusercontent.com/79876959/141163473-8597c6a3-32b2-4327-b5c1-0bd6a79bef20.PNG)